zanata-war/src/main/java/org/zanata/service/TranslationStateCache.java

```diff
@@ -90,4 +91,6 @@ WordStatistic getDocumentStatistics(Long documentId,
      * @param localeId
      */
     void clearDocumentStatistics(Long documentId, LocaleId localeId);
+
+    void docStatsUpdated(DocStatsEvent event);
```

Comment:
Is this just declaring one of the `@Observes` event handler methods? I don't really feel comfortable with exposing these event handlers in the interface signature, because the methods are supposed to be invoked by CDI as an implementation detail, not directly by code as part of the public interface. Does this one need to be here?